src/ElmSharp/ElmSharp/PreloadedWindow.cs

```diff
@@ -25,6 +25,9 @@ public PreloadedWindow(bool useBaseLayout=true) : base("PreloadWindow")
             s_precreated = this;
             if (useBaseLayout)
                 InitializeBaseLayout();
+            CreateWidgets();
+            BackButtonPressed += DummyHandler;
```

Comment:
How about use local function? 
https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/classes-and-structs/local-functions

Reply:
You mean to add/remove event in the method like `CreateWidgets()`? Ok. Sounds coherent.

Reply:
Done